Composer/packages/tools/language-servers/language-generation/src/LGServer.ts

```diff
@@ -496,6 +502,33 @@ export class LGServer {
       return Promise.resolve(null);
     }
 
+    const wordRange = getEntityRangeAtPosition(document, params.position);
+    const word = document.getText(wordRange);
+    const projectId = this.getLGDocument(document)?.projectId;
+    let suggestEntities: string[] = [];
+    if (projectId && this.entitiesResolver) {
+      const luConetnts = this.entitiesResolver(projectId);
+      if (luConetnts) {
+        for (const conetnt of luConetnts) {
+          const luisJson = await extractLUISContent(conetnt);
+          suggestEntities = suggestEntities.concat(getSuggestionEntities(luisJson, suggestionAllEntityTypes));
+        }
+      }
+
+      this.luisEntities = suggestEntities;
+    }
+
+    const startWithAt = word.startsWith('@');
+
+    const completionEntityList = this.luisEntities.map((entity: string) => {
+      return {
+        label: entity,
+        kind: CompletionItemKind.Keyword,
```

Comment:
I dont think this should be keyword, it will cause a certain ui treatment on the monaco side.
keyword usually is used for language grammar keywords such as if else, for loop, etc.

Reply:
Switch to CompletionItemKind.Property. How about this? Since it does not have a explicit mapping of entity to CompletionItemKind